### PR TITLE
Document "Reference/Commands/x-update-registry"

### PR DESCRIPTION
**Pull request by BillyONeal**

### CODE REVIEW

File: vcpkg/commands/update-registry.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+## Synopsis
+
+```console
+vcpkg x-update-registry [--all] artifact-registry-name-or-uri...
+```
+
+## Description
+
+Downloads the registry contents of the supplied artifact registry.
```
Comment (AugP) [inline vcpkg/commands/update-registry.md:18]: Add a bit of context on when the user would interact with this feature. It's not clear from this sentence why the user would run this command. What is the "problem" the user is trying to solve by running this? Imagine you're the user and you run an online search for your problem. Reading this description should clarify that they found the right page with the solution. 

Also add a "This feature does not apply to vcpkg port registries" warning.

**Review by AugP — CHANGES_REQUESTED** [review]

File: vcpkg/commands/update-registry.md (hunk 1 above)
Comment (BillyONeal) [reply vcpkg/commands/update-registry.md:18]: I'm not sure 'describe a scenario where you want to use this' is appropriate for reference level 'this is what all the switches do' documentation like this. For that, unfortunately at this time there are only embedded use cases for which I do not have great understanding at this time.

How do you want the warning worded? It already says 'artifact registry'.

**Comment by AugP** [comment]
PR converted to draft as we are pausing artifact documentation for the time being.

**Comment by BillyONeal** [comment]
> PR converted to draft as we are pausing artifact documentation for the time being.

This is the new docs added at the same time as a new feature, I think now that it is written it should be added.

**Comment by vicroms** [comment]
Depends on basic documentation about artifacts that we don't have, drafting until we have that.

**Pull request closed without merging**